components/templates/ArticleTemplate/ArticleTemplateStyles.ts

```diff
@@ -8,8 +8,15 @@ export const StyledHeaderImage = styled.img`
 `
 export const StyledAuthorByline = styled.p`
     color: #6c757d;
+    margin: 0 0 0.5rem;
+`
+
+export const StyledDates = styled.text`
+    color: #6c757d;
```

Comment:
You know, I have this thought about the author byline as well — should we go with a darker color for both? Not setting it will default to `#212529`, which looks like it may be slightly more readable?
![Screen Shot 2021-09-03 at 1 14 08 PM](https://user-images.githubusercontent.com/43454804/132043344-42b0f86c-c298-4f56-bc44-8b4e31a6bd97.png)

Reply:
I took the color out and now it is default. I guess if we eventually have clickable authors we may want a different color?